ingestion/setup.py

```diff
@@ -213,7 +214,8 @@ def get_long_description():
     "redash": {VERSIONS["packaging"]},
     "redpanda": {*COMMONS["kafka"]},
     "redshift": {
-        "sqlalchemy-redshift~=0.8",
+        # Going higher has memory and performance issues
+        "sqlalchemy-redshift==0.8.12",
```

Comment:
@pmbrull did we open an issue on the `sqlalchemy-redshift` repo by any chance?

Reply:
yeah on my todo list, let me do it right away

Reply:
https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/282